src/client/Microsoft.Identity.Client/MsalError.cs

```diff
@@ -908,5 +908,15 @@ public static class MsalError
         public const string FailedToCreateSharedPreference = "shared_preference_creation_failed";
 
 #endif
+
+        /// <summary>
+        /// When calling AcquireTokenInteractive with the WAM broker, the call must be made from the UI thread.
+        /// </summary>
+        public const string WamUiThread = "wam_ui_thread_only";
+
+        /// <summary>
+        /// The Windows broker (WAM) is not supported in conjunction with B2C authorities
+        /// </summary>
+        public const string WamNoB2C = "wam_no_b2c";
```

Comment:
>WamNoB2C [](start = 28, length = 8)

nit: we're trying to move away from "b2c" branding in the api...is there a different way to say it w/out b2c? `WamNotSupported` or something? Assume you'll have an aka.ms link somewhere w/more details.

Reply:
ack

Reply:
I will use the Windows terminology, smth like "The Windows broker (WAM) is only supported in conjunction with "work and school" accounts and with Microsoft accounts."

Reply:
good idea.